src/server/lib/passcodeSendAgainTimer.ts

```diff
@@ -0,0 +1,15 @@
+import { Request } from 'express';
+
+export const getPasscodeSendAgainTimer = (req: Request): number => {
+	// if we're running in cypress, we want to use the cypress mock state cookie to set the timer for testing
+	const runningInCypress = process.env.RUNNING_IN_CYPRESS === 'true';
+	if (runningInCypress) {
+		const cypressMockStateCookie = req.cookies['cypress-mock-state'];
```

Comment:
maybe a silly question because I don't know cypress that well:

is there any reason for this to be called `cypress-mock-state` and not something that refers to the timer ?

Reply:
It's because it's a pattern we use already in Gateway to customise behaviour on the sever based on Cypress tests.

So we set a `cypress-mock-state` cookie in other tests that also modifies behaviour is other ways only when it exists. Which is useful for testing edge case scenarios.